Formula/dsh.rb

```diff
@@ -24,7 +24,15 @@ class Dsh < Formula
 
   depends_on "libdshconfig"
 
+  on_macos do
+    depends_on "gnu-sed" => :build
+  end
+
   def install
+    # Use GNU sed on macOS to avoid this build failure:
+    # sed: RE error: illegal byte sequence
+    ENV.prepend_path "PATH", Formula["gnu-sed"].libexec/"gnubin" if OS.mac?
```

Comment:
Has this been reported upstream?

Reply:
There is no upstream bug tracker but I reported this by email to the upstream developer, and mentioned this in the comment.